welcomescreen.py

```diff
@@ -229,7 +229,7 @@ def get_sonic_pos():
             if help_btn.collidepoint(mos_x, mos_y) and \
                (pygame.mouse.get_pressed())[0] == 1 and \
                press == 0 and not self.showing_help:
-                self.showing_help = True
+                    self.showing_help = True
```

Comment:
This is supposed to be the block under the if condition and it's way more than 4 spaces.

Reply:
Removed this commit through force push